apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -200,6 +201,19 @@ const PaymentsTableContent: React.FC<IPaymentsTableProps> = ({ billingContext, c
         }
     }, [])
 
+    /**
+    * set default filters only when filters are empty and reset button hasn't been pressed
+    **/
+    useEffect(()=>{
+        if (hasFilters){
+            filtersAreReset = false
+        }
+        if (!hasFilters && !filtersAreReset) {
+            isDefaultFilterApplied = true
+            setDateRange(DEFAULT_DATE_RANGE)
+        }
+    }, [hasFilters])
```

Comment:
filtersAreReset

Reply:
I only want this `useEffect` to run when `hasFilters` changed